src/models/comments_model.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose, { Document, Schema, ObjectId } from "mongoose";
+
+export interface ICommentDocument extends Document {
```

Comment:
```suggestion
export interface IComment {
```
From what I see, `Document` is an actual mongoose object on which you can perform actions. That's not what this interface represents.